l2geth/params/config.go

```diff
@@ -437,6 +467,23 @@ func (c *ChainConfig) IsSDUpdate(num *big.Int) bool {
 	return true
 }
 
+// To check whether Fee Token Update or not
+func (c *ChainConfig) IsFeeTokenUpdate(num *big.Int) bool {
+	if c.ChainID == nil {
+		return true
```

Comment:
I think it have to become false! 🙇🏼‍♂️

Reply:
Thanks but I assumed this code will provide in production after fee token update. BTW, Do you know the case that 'chain id is nil'? I think this code section should be removed if we don't have to check it.

Reply:
Okay. I agree your comments. 
But I think, It is not natural to assume to use FeeToken(Ton). Basically, who use our l2geth thinks to pay by ether.